src/presentation/http/router/auth.ts

```diff
@@ -53,12 +58,19 @@ const AuthRouter: FastifyPluginCallback<AuthRouterOptions> = (fastify, opts, don
     const accessToken = opts.authService.signAccessToken({ id: userSession.userId });
 
     await opts.authService.removeSessionByRefreshToken(token);
-    const refreshToken = await opts.authService.signRefreshToken(userSession.userId);
+    const { refreshToken, expiresAt } = await opts.authService.signRefreshToken(userSession.userId);
 
-    return reply.send({
-      accessToken,
-      refreshToken,
-    });
+    return reply
+      .setCookie('refreshToken', refreshToken, {
```

Comment:
maybe call it "rt" ?

Reply:
I think this naming is not clear enough

Reply:
camel case is not a common practice for cookie names, as I know

Reply:
yep